packages/cozy-client/src/cli/index.js

```diff
@@ -192,4 +192,22 @@ const createClientInteractive = (clientOptions, serverOpts) => {
   })
 }
 
+const main = async () => {
+  const client = await createClientInteractive({
+    scope: ['io.cozy.files'],
+    uri: 'http://cozy.tools:8080',
+    oauth: {
+      softwareID: 'io.cozy.client.cli'
+    }
+  })
+  console.log(client.toJSON())
+}
+
+if (require.main === module) {
+  main().catch(e => {
+    console.error(e)
+    process.exit(1)
+  })
+}
+
```

Comment:
Je comprends pas trop le message du commit par rapport au code ?

Reply:
This commits adds a way to execute the cli/index.js, to test the createClientInteractive method.

Reply:
Aaaah, makes sens now! Thanks